app/src/main/java/org/mozilla/tv/firefox/webrender/VideoVoiceCommandMediaSession.kt

```diff
@@ -171,7 +171,7 @@ class VideoVoiceCommandMediaSession @UiThread constructor(
         //
         // The videos may send playback state update events to Java, which we're forced to ignore:
         // see JavascriptVideoPlaybackStateSyncer for the code.
-        webView?.evalJS("document.querySelectorAll('video').forEach(v => v.pause());")
+        engineView?.evalJS("document.querySelectorAll('video').forEach(v => v.pause());")
```

Comment:
huh, I didn't realize that but I guess the engineView can call evalJS.

Reply:
Well it's an Engine ext method FWIW